Source/ConstraintMakerPriortizable.swift

```diff
@@ -30,38 +30,90 @@
 
 public class ConstraintMakerPriortizable: ConstraintMakerFinalizable {
     
+    public enum Priority: Strideable, RawRepresentable, Equatable {
```

Comment:
Oh, IMHO this enum should not be inside ConstraintMakerPriortizable.
We don't want to write ConstraintMakerPriortizable.Priority right?